server.lua

```diff
@@ -1,8 +1,57 @@
-local QBCore
+local Core, CoreName
 if GetResourceState('qb-core') == 'started' then
-    QBCore = exports['qb-core']:GetCoreObject()
+    Core = exports['qb-core']:GetCoreObject()
+    CoreName = 'qb'
+elseif GetResourceState('es_extended') == 'started' then
+    Core = exports["es_extended"]:getSharedObject()
+    CoreName = 'esx'
 else
-    warn('qb-core is missing, modifications won\'t cost anything')
+    print('Core is missing nothing will cost')
+end
+Functions = {}
+
+Functions.qb = {}
+Functions.qb.GetPlayer = function(src)
+    return Core.Functions.GetPlayer(src)
+end
+Functions.qb.GetMoney = function(player, type)
+    return player.Functions.GetMoney(type)
+end
+Functions.qb.RemoveMoney = function(player, type, amount)
+    return player.Functions.RemoveMoney(type, amount, "Customs")
+end
+Functions.qb.GetJob = function(player)
+    return player.PlayerData.job.name 
+end
+Functions.qb.Owned = function(vehicleplate)
+    local result = MySQL.salar.await('SELECT 1 from player_vehicles WHERE plate = ?', {plate})
+    return result
+end
+Functions.qb.Update = function(vehicleplate)
+    MySQL.update('UPDATE player_vehicles SET mods = ? WHERE plate = ?', {json.encode(vehicleProps), vehicleProps.plate})
+end
+Functions.esx = {}
+Functions.esx.GetPlayer = function(src)
+   return Core.GetPlayerFromId(src) 
+end
+Functions.esx.GetCash = function(player)
+    return player.getMoney()
+end
+Functions.esx.GetBank = function(player)
+    return player.getAccount('bank').money
+end
+Functions.esx.RemoveMoney = function(player, type, amount)
+    return player.removeAccountMoney(type, amount)
+end
+Functions.esx.GetJob = function(player)
+    return player.getJob().name

```

Comment:
It's a good practice to use the safe operator of `?` for nil checking to reduce code error. Same thing applies to GetBank()